projects/mac_ai/testing/prepare_mac_ai.py

```
@@ -48,6 +42,33 @@ def cleanup():
         if is_running:
             prepare_llama_cpp.cleanup_image(base_work_dir)
 
+    # ***
+
+    if config.project.get_config(f"cleanup.files.ollama"):
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)